client/my-sites/subscribers/components/grow-your-audience/grow-your-audience.tsx

```diff
@@ -1,57 +1,101 @@
-				<a className="grow-your-audience__card-link" href={ url } target="_blank" rel="noreferrer">
-					{ translate( 'Learn more' ) }
+const GrowYourAudienceCard = ( {
+	ctaLabel,
+	externalUrl,
+	icon,
+	text,
+	title,
+	url,
+}: GrowYourAudienceCardProps ) => (
+	<Card className="grow-your-audience__card" size="small">
+		<CardBody className="grow-your-audience__card-body">
+			<h3 className="grow-your-audience__card-title">
+				<Icon className="grow-your-audience__card-icon" icon={ icon } size={ 20 } />
+				{ title }
+			</h3>
+			<p className="grow-your-audience__card-text">{ text }</p>
+			{ externalUrl ? (
+				<ExternalLink className="grow-your-audience__card-link" href={ url }>
+					{ ctaLabel }
+				</ExternalLink>
+			) : (
+				<a className="grow-your-audience__card-link" href={ url }>
+					{ ctaLabel }
 				</a>
-			</CardBody>
-		</Card>
-	);
-};
+			) }
+		</CardBody>
+	</Card>
+);
 
 const GrowYourAudience = () => {
+	const locale = useLocale();
 	const translate = useTranslate();
 	const selectedSiteSlug = useSelector( getSelectedSiteSlug );
 
+	const statsCardTranslated =
+		englishLocales.includes( locale ) ||
+		( hasTranslation( 'Explore your stats' ) &&
+			hasTranslation(
+				'Take a look at your stats and refine your content strategy for better engagement.'
+			) &&
+			hasTranslation( 'Check stats' ) );
+
 	return (
 		<SectionContainer className="grow-your-audience">
 			<h2 className="grow-your-audience__title">{ translate( 'Grow your audience' ) }</h2>
 
 			<div className="grow-your-audience__cards">
+				{ statsCardTranslated ? (
+					<GrowYourAudienceCard
+						icon={ chartBar }
+						text={ translate(
+							'Take a look at your stats and refine your content strategy for better engagement.'
+						) }
+						title={ translate( 'Explore your stats' ) }
+						ctaLabel={ translate( 'Check stats' ) }
+						url={ `/stats/subscribers/${ selectedSiteSlug }` }
+					/>
+				) : (
+					<GrowYourAudienceCard
+						icon={ chartBar }
+						text={ translate(
+							'Using a subscriber block is the first step to growing your audience.'
+						) }
+						title={ translate( 'Every visitor is a potential subscriber' ) }
+						ctaLabel={ translate( 'Learn more' ) }
+						externalUrl
+						url={ localizeUrl(
+							'https://wordpress.com/support/wordpress-editor/blocks/subscribe-block/'
+						) }
+					/>
+				) }
+
 				<GrowYourAudienceCard
-					icon={ chartBar }
+					icon={ trendingUp }
 					text={ translate(
-						'Using a subscriber block is the first step to growing your audience.'
-					) }
-					title={ translate( 'Every visitor is a potential subscriber' ) }
-					url={ localizeUrl(
-						'https://wordpress.com/support/wordpress-editor/blocks/subscribe-block/'
+						'Allow your readers to support your work with paid subscriptions, gated content, or tips.'
 					) }
+					title={ translate( 'Start earning' ) }
+					ctaLabel={ translate( 'Learn more' ) }
```

Comment:
Wonder if this ctaLabel should be something different than 'Learn more' now that it just links directly to the Earn page?

Reply:
I think so! Any ideas? :-)

Reply:
Not sure. We can just choose something - it's easy enough to change later. 

Some options:
Start earning
Explore earn options
Explore monetization
Go to Earn
Go to Earn page
Visit Earn dashboard
See earn options
See monetization options